packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/TokenInputSection/QuickAmountSelector.tsx

```
@@ -1,24 +1,49 @@
+            flex={1}
+            size="medium"
+            variant="secondary"
+            borderTopRightRadius={index !== amountsLength - 1 ? 0 : '$2'}
+            borderBottomRightRadius={index !== amountsLength - 1 ? 0 : '$2'}
+            borderTopLeftRadius={index !== 0 ? 0 : '$2'}
+            borderBottomLeftRadius={index !== 0 ? 0 : '$2'}
+            onPress={() => onSelect(amount.value.toString())}
+          >
+            {amount.label}
+          </Button>
+          {index !== amountsLength - 1 ? <Divider vertical /> : null}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)